tests/queries/0_stateless/02981_system_dns_cache_table.sql

```diff
@@ -0,0 +1 @@
+select hostname, ip_address, ip_family, (isNotNull(cached_at) and cached_at > '1970-01-01 00:00:00') from system.dns_cache
```

Comment:
`isNotNull(cached_at)` unnecessary

Reply:
We use the same ClickHouse server for multiple stateless tests. There will likely be other cache entries at this point. The best solution is to make a request to `localhost` explicitly and then search for the entries in `system.dns_cache`.

Reply:
> `isNotNull(cached_at)` unnecessary

No, it's a check for caching time saved properly

Reply:
> There will likely be other cache entries at this point.

thx, it's fixed